### PR TITLE
Errata from ch7

### PR DESCRIPTION
**Pull request by omundy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,5 +83,41 @@ Credit: [@omundy](https://github.com/omundy)
 With `InventoryObject` selected, in the Canvas Scaler component, set `Reference Pixels Per Unit` to 32.
 
 
+#### Chapter 7, Page 263
+Credit: [@omundy](https://github.com/omundy)
+
+In the **The DamageCharacter() method** section, the code should be added to the `Enemy` class.
```
Comment (JaredHalpern) [inline README.md]: Hey Owen - I think the text already specifies that this should be added to the `Enemy` class, unless I'm misunderstanding something here.

File: README.md [hunk 2]
```diff
@@ -83,5 +83,41 @@ Credit: [@omundy](https://github.com/omundy)
...
+```
+private void OnEnable() {
+	ResetCharacter();
+}
+```
+
+#### Chapter 7, Page 275
+Credit: [@omundy](https://github.com/omundy)
+
+You'll notice when you run into the enemies they spin. Like you did with the PlayerObject, check `Freeze Rotation` for the Z axis inside the Rigidbody2D on the EnemyObject prefab.
```
Comment (JaredHalpern) [inline README.md]: Oomph - good catch!

File: README.md (hunk 1 above)
Comment (omundy) [reply README.md]: The preceding section heading has the word Enemy Class in it, but then describes the Character class (and the player and enemy), so by the next section it wasn't not obvious to me. Many other code blocks that begin with "Add the following..." explicitly mention the class name, which I think is helpful.

File: README.md (hunk 2 above)
Comment (omundy) [reply README.md]: *Actually kind of funny to see them scatter like bowling pins. I think it's good when a reader is allowed to make small errors and have a moment to think about what the problem may be *before* the text guides them.

File: README.md (hunk 1 above)
Comment (JaredHalpern) [reply README.md]: Ah, I see. Yes, I agree - clarification is a good idea here. Thanks for pointing it out.

File: README.md (hunk 2 above)
Comment (JaredHalpern) [reply README.md]: Agreed. Personally, I learn best by breaking things then taking a moment to think about it. The first time the player runs into a collider (page 134), the `Freeze Rotation Z` box isn't checked, and they can see the weird results.

If I do a 2nd edition of this book, perhaps I'll make a point of instructing the reader to pause and consider *why* something is broken.

File: Chapter 7/Enemy.cs [hunk 3]
```diff
@@ -25,8 +25,6 @@ public override IEnumerator DamageCharacter(int damage, float interval)
         {
             hitPoints = hitPoints - damage;
 
-            StartCoroutine(FlickerCharacter());
```
Comment (JaredHalpern) [inline Chapter 7/Enemy.cs]: Ah yes - I know exactly how this was introduced too. *kicks self* thank you for pointing out.

File: README.md [hunk 4]
```diff
@@ -83,5 +83,41 @@ Credit: [@omundy](https://github.com/omundy)
...
+
+
+
+#### Chapter 8, Page 287
+Credit: [@omundy](https://github.com/omundy)
+
+In the **Enemy Walk Animation** section, select the Enemy prefab then open the **Animator** window as seen in Figure 8-3.
```
Comment (JaredHalpern) [inline README.md]: You have a great eye for detail. Thank you.

**Pull request merged**